takwimu/templates/takwimu/_includes/profile/topic_detail.html

```diff
@@ -4,7 +4,7 @@
   <div class="container py-5">
     <div class="row">
       <!-- Left navigation of all indicators-->
-      <div class="col-3 dataindicators-nav">
+      <div class="col-4 dataindicators-nav sticky-top d-table" style="top: 100px;">
```

Comment:
Why the `d-table`?

Reply:
@DavidLemayian 
I need the left menu to have fluid height depending on the content, So that `sticky-top` class
will behave accordingly.

Reply:
Sounds good. Thanks.